pyproject.toml

```diff
@@ -47,6 +47,7 @@ dependencies = [
   'uwsgi-readiness-check~=0.2.0',
   'django-allow-cidr',
   'django-csp~=3.7',
+  'django-ansible-base[jwt-consumer,resource-registry] @ git+https://github.com/ansible/django-ansible-base@devel',
```

Comment:
EDA (that we tend to follow in some respects) adds it like this:

https://github.com/ansible/eda-server/blob/main/pyproject.toml#L51C1-L59C4

```
django-ansible-base = { git = "https://github.com/ansible/django-ansible-base.git", tag = "2025.1.31", extras = [
    ...
    "resource-registry",
    "jwt-consumer",
] }
```
I'd have thought using a concrete version was preferable over the `devel` branch?